### ISSUE TITLE
checkConfig add mysql privilege check

### ISSUE THREAD

**Issue by martianzhang** [XiaoMi/soar#126]
#### Feature Description

OnlineDSN need select privielge
TestDSN need all privilege on optimizer_xx.*

#### Use Case(s)

```bash
soar -check-config
```

if privilege not match give suggest

**Comment by martianzhang** [comment]
```sql
select current_user()
```

* OnlineDSN
```sql
select Select_priv from mysql.user where user='root' and host='%';
```

* TestDSN
```sql
SELECT GROUP_CONCAT(COLUMN_NAME) from information_schema.COLUMNS where TABLE_SCHEMA='mysql' and TABLE_NAME='user' and COLUMN_NAME like "%_priv";

select %_priv from mysql.user where user='root' and host='%';
```

**Comment by martianzhang** [comment]
duplicate with #91 , fix this issue #91 will also be fixed.
